changelog.md

```diff
@@ -6,6 +6,10 @@
 - react-native-bpk-component-ticket:
   - New component, see http://backpack.prod.aws.skyscnr.com/components/native/ticket
 
+- react-native-bpk-component-animate-height:
+  - Added props for `expandDelay` and `collapseDelay`.
+  - Now removes itself from tree when collapsed to height 0.
```

Comment:
Sorry to be pedantic, but I think we can make this more meaningful for users, something like "when collapsed to `height: 0`, or "when collapsed and height is zero". What do you think?

Reply:
Yeah sure. Also, never apologise for being pedantic 😆